oletools/olevba.py

```diff
@@ -227,7 +227,7 @@
 
 import sys, logging
 import struct
-import cStringIO
+from io import StringIO
```

Comment:
for now I think that change might break python 2.x compatibility, because cStringIO is used to handle bytes, not unicode. This is why we still have a separate olevba3 script for python 3. I plan to merge olevba with olevba3 in the near future, but it's significant work to make sure all the bytes and str/unicode are correctly used.